backend/service/status/status_service.go

```diff
@@ -56,6 +56,7 @@ func (s *Service) RegisterRoute(r rest.Router) {
 	r.GET(v1.PathForStatus, s.serverStatus())
 	r.GET(v1.PathForStatusNode, s.nodeStatus())
 	r.GET("/v1/metrics", s.metricsHandler())
```

Comment:
replace /v1/metrics with v1.PathForMetrics.

also what is the difference between /v1/metrics and /metrics and why have both?

Reply:
`/metrics` is for Prometheus.
the reason it is not part of `v1` because it's internal.